Changelog.md

```diff
@@ -8,7 +8,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- Add `VK_KHR_present_wait` extension (#493)
+- Added `VK_KHR_present_wait` device extension (#493)
+
+### Changed
+
+- Device extension `ext::ExtendedDynamicState` now exposes `fn device()` instead of `fn instance()` (#494)
+- Device extension `khr::PushDescriptor` now exposes `fn device()` instead of `fn instance()` (#494)
+- Device extension `khr::PipelineExecutableProperties` now exposes `fn device()` instead of `fn instance()` (#499)
+- Changed `khr::PipelineExecutableProperties::new()` to take `instance` and `device` as arguments (#499)
+- Device extension `khr::TimelineSemaphore` now exposes `fn device()` instead of `fn instance()` (#499)
+- Changed `khr::TimelineSemaphore::new()` to take `instance` and `device` as arguments (#499)
+
+### Removed
+
+- Removed `device` argument from `ext::DebugMarkers::debug_marker_set_object_name` function (#494)
+- Removed `From<vk::Result>` trait for `VkResult` (#495)
+- Removed `instance` argument from `ext::DebugUtils::submit_debug_utils_message` function (#499)
+- Removed `device` argument from `khr::PipelineExecutableProperties` functions (#499)
+- Removed `device` argument from `khr::TimelineSemaphore` functions (#499)
```

Comment:
Thanks, also for taking care of #499. Just not sure if we should mention some single PRs this often in the changelog.

For example, the `### Removed` section is afaik for removed API - changing the arguments to remove ie. `device` that's already available in the struct is already documented under `### Changed`, no need to repeat that again.

Furthermore, we can probably make lists, ie turn this:

```markdown
- Device extension `ext::ExtendedDynamicState` now exposes `fn device()` instead of `fn instance()` (#494)
- Device extension `khr::PushDescriptor` now exposes `fn device()` instead of `fn instance()` (#494)
- Device extension `khr::PipelineExecutableProperties` now exposes `fn device()` instead of `fn instance()` (#499)
```

into:

```markdown
- Device extensions `ext::ExtendedDynamicState`, `khr::PushDescriptor` and `khr::PipelineExecutableProperties` now expose `fn device()` instead of `fn instance()` (#494, #499)
```

I think that's much more more concise (more readable).

Reply:
@Ralith What are your thoughts on this, and this PR in general? Seems to be pretty much ready to go in besides some changelog bikeshedding, unless you have some burning comments?

Would like to get this in and do a release soon, it has been way too long.

Reply:
Your proposed tweaks to the changelog SGTM, and overall this looks like great cleanup, though I haven't reviewed every single change in detail.

Reply:
The changes are pretty boring. It's mostly code gardening.

Reply:
@filnet Yes, curating the changelog (usually when preparing for a release) is pretty tough. We usually end up going for just the commit titles since the last release, but that's usually not fully representative nor are all the titles good enough for direct use :)